vendor/assets/javascripts/jquery.adaptivemenu.js

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Diff with original and add some comments around intent and how that's carried out below.